google/cloud/gkehub/BUILD.bazel

```diff
@@ -27,12 +27,12 @@ MOCK_HEADER_GLOB = "mocks/*.h"
 cc_library(
     name = "google_cloud_cpp_gkehub",
     srcs = glob(
-        include=[SOURCE_GLOB],
-        exclude=[MOCK_SOURCE_GLOB],
+        include = [SOURCE_GLOB],
```

Comment:
Just FYI (and for future reference): seems like you skipped the formatting step after generating the code.